velox/connectors/hive/storage_adapters/abfs/AbfsConfig.h

```diff
@@ -92,6 +93,12 @@ class AbfsConfig {
     return authorityHost_;
   }
 
+  /// Test only.
+  static void registerFakeWriteFileClient(
+      std::unique_ptr<AzureDataLakeFileClient> fakeClient) {
+    fakeWriteClient_ = std::move(fakeClient);
```

Comment:
For safety and simplicity, let's check and initialize the singleton `testWriteClient_` here.